### DISCUSSION TITLE
RAC Table, recommended way to create frozen columns?

### DISCUSSION THREAD

**Discussion by TENsaga** [adobe/react-spectrum#6081] [Q&A]
Looking to add a left side frozen column with box shadow border (on the right of the column), so the data table content can scroll under it and the frozen column remains in focus. 

https://react-spectrum.adobe.com/react-aria/Table.html

**Comment by snowystinger** [comment]
Could you do something like this? https://css-tricks.com/a-table-with-both-a-sticky-header-and-a-sticky-first-column/

**Reply by TENsaga** [reply]
Thanks, will take a look once I dive into this again.
